src/main/java/com/semmle/jira/addon/config/ProcessedConfig.java

```diff
@@ -41,8 +41,13 @@ public ProcessedConfig(Config config) throws InvalidConfigurationException {
 
     if (config.getIssueTypeId() != null) {
       taskIssueType = ComponentAccessor.getConstantsManager().getIssueType(config.getIssueTypeId());
-      if (taskIssueType == null)
+      if (taskIssueType == null) {
         configErrors.add(String.format("issue type id '%s' not found", config.getIssueTypeId()));
+      } else if (!taskIssueType.getName().toLowerCase().equals("LGTM alert".toLowerCase())) {
+        configErrors.add("issue type is not 'LGTM alert'");
+      } else if (!project.getIssueTypes().stream().anyMatch(taskIssueType::equals)) {
```

Comment:
This check its wrong, as it will only try it if the issue type is the correct one. Also, we should not check this here, it is good enough to say `issue type is not 'LGTM alert`.

Reply:
After correct configuration, if the project issue scheme is later edited to remove the `LGTM alert` type, doesn't this catch a legitimate error condition? Or are you saying that (in that case) the user will get sufficient errors/logs when transitions are tried but fail?

Reply:
Also, I think it's valid to make this check only if the issue type is `LGTM alert`. If the add-on isn't configured to use the correct `IssueType` then not being part of the issue scheme isn't relevant information.

Reply:
Yes, that can be a valid check, but it is also being made in the js when setting the configuration. And for removing the issue type you have to migrate the existing issues.